test/com/yetu/youtube/utils/FakeGlobal.scala

```diff
@@ -41,7 +41,7 @@ object FakeGlobal {
    * An identity.
    */
   val identity = User(
-    userID = UUID.randomUUID(),
+    userUUID = UUID.randomUUID().toString,
```

Comment:
good point, I'll update the model for this. I'll do this in the extracted library though.